gateways/class-wc-ebanx-gateway.php

```diff
@@ -98,16 +98,63 @@ public function __construct() {
 		$this->merchant_currency = strtoupper( get_woocommerce_currency() );
 	}
 
+	protected function debug_log_if_available($message, $event = null, $persist = false) {
```

Comment:
This function has some breakpoints that could be checked at the beginning of it, it would be a good idea to centralize them

Reply:
Is centralized. I really didn't realize the point here.